src/router/index.js

```diff
@@ -1,43 +1,43 @@
 'use strict';
 
 const express = require('express');
-const db = require('./db');
+const db = require('./../db');
+const productsRoutes = require('./products');
 
 const router = express.Router();
 
+router.get('/', productsRoutes.allProducts);
+
 router.post('/register', (req, res, next) => {
+  console.log("someone is trying to register");
+  db.register(req.body);
   res.sendStatus(200)
     .next();
-  //const newUser = {
-  //  userID: req.body.id,
-  //  passwordHash: req.body.hash,
-  //};
-  //write new user to db asyncronously
-  //.then response.status(200)
-  //.catch
 });
 
 router.post('/login', (req, res, next) => {
   res.sendStatus(200)
     .next();
```

Comment:
I am curious about .next() and what it does. Please let me know.

Reply:
The next function tells your server to call the next middleware.